### ISSUE TITLE
URL encoding in custom logic block

### ISSUE THREAD

**Issue by GitHubber1** [marcelotto/spread2rdf#8]
Hello Marcel,

so far I use own logic blocks to generate URL encoded URIs. As maybe some other people will find this helpful I decided to post it here so that you can include this snippet in an example mapping file if you wish.

```
column  :anycolumn, statement: :none do
          encoded_value = URI::encode(value)
          encoded_URI = RDF::URI.new("#{YOURNAMESPACE}#{encoded_value}")
          statement(subject, YOURNAMESPACE.example, encoded_URI )
end
```

Cheers,
Andy

**Comment by marcelotto** [comment]
You're right, this might be so common, I'd like to make this the default, so that the following will do the same as your custom logic block:

``` ruby
column :anycolumn, predicate: YOURNAMESPACE.example, 
                   object: { uri: { namespace: YOURNAMESPACE } }
```

Currently, this performs a custom [URI normalization](http://github.com/marcelotto/spread2rdf/blob/master/lib/spread2rdf/mapping/default_cell_mappings.rb), which was used in the context of the project Spread2RDF was developed for, but a standard URI encoding, would be a better method for the standard case. However, first I would like to find a better way to switch the normalization method used by URI construction with a specified namespace.
